medusa/config.py

```diff
@@ -75,13 +82,26 @@
     ['enabled', 'cassandra_url', 'use_mgmt_api']
 )
 
+CONFIG_SECTIONS = {
```

Comment:
I guess this requires a rebase on top of `master`  now that your other PR was merged. It will avoid to duplicate changes in different commits.